### PR TITLE
Tab Completion for /pv & /cata, typo, and cleanup

### PR DESCRIPTION
**Pull request by walkerselby**
Added IGN tab completion for /pv and /cata
Fixed a typo in /help
Deleted unused commented code
Deleted duplicate lines of code related to DIVINE
Deleted Tutorial -- it is unused and can always be found in git later if needed. Better than just having it all sitting commented.

### CODE REVIEW

**Comment by Ascynx** [comment]
scared

File: src/main/java/io/github/moulberry/notenoughupdates/commands/profile/CataCommand.java [hunk 1]
```diff
@@ -33,4 +39,18 @@ public void processCommand(ICommandSender sender, String[] args) {
 		GuiProfileViewer.currentPage = GuiProfileViewer.ProfileViewerPage.DUNGEON;
 		super.processCommand(sender, args);
 	}
+	@Override
+	public List<String> addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/profile/CataCommand.java:43]: Doesn't CataCommand already extend ViewProfileCommand, which already implements this sort of tab completion.

Comment (walkerselby) [reply src/main/java/io/github/moulberry/notenoughupdates/commands/profile/CataCommand.java:43]: I think you are right, will remove.

**Review by walkerselby — COMMENTED** [review]
No longer going to be a part of 2.1

**Comment by walkerselby** [comment]
Going to revise and then re-PR this soon, found some issues.

**Pull request closed without merging**